Indicorp/index.html

```diff
@@ -0,0 +1,385 @@
+                        <iframe width="597" height="442" src="https://www.youtube.com/embed/p_SaeH1qpng" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+                        <button class="play__btn"></button>
+                    </div>
+
+                    <div class="info">
+                        <div class="info__title">
+                            <h2>Most trusted in our field</h2>
+                            <p>Most calendars are designed for teams. Slate is designed for freelancers who want a simple way to plan their schedule.</p>
+                        </div>
+
+                        <div class="info__card card1">
+                            <img src="./img/people.svg" alt="people">
+
+                            <div class="card__info">
+                                <h5>the quick fox jumps over the lazy dog</h5>
+                                <h6>Things on a very small scale ...</h6>
+                            </div>
+                        </div>
+
+                        <div class="info__card">
+                            <img src="./img/time.svg" alt="people">
+
+                            <div class="card__info">
+                                <h5>the quick fox jumps over the lazy dog</h5>
+                                <h6>Things on a very small scale ...</h6>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </section>
+
+        <!-- Clients section -->
+        <section id="clients" class="clients">
+            <div class="container">
+                <ul class="clients__list">
+                    <li>
+                        <a href="http://hooli.com/"><img src="./img/fa-brands_hooli.svg" alt="hooli"></a>
+                    </li>
+                    <li>
+                        <a href="https://www.lyft.com/"><img src="./img/fa-brands_lyft.svg" alt="lyft"></a>
+                    </li>
+                    <li>
+                        <a href="http://www.piedpiper.com/"><img src="./img/fa-brands_pied-piper-hat.svg" alt="pied-piper"></a>
+                    </li>
+                    <li>
+                        <a href="https://stripe.com/"><img src="./img/fa-brands_stripe.svg" alt="stripe"></a>
+                    </li>
+                    <li>
+                        <a href="https://aws.amazon.com/"><img src="./img/fa-brands_aws.svg" alt="aws-amazon"></a>
+                    </li>
+                    <li>
+                        <a href="https://www.reddit.com/"><img src="./img/fa-brands_reddit-alien.svg" alt="reddit"></a>
+                    </li>
+                </ul>
+
+                <div class="subscribe">
+                    <form class="subscribe__form">
+                        <label for="email"><h3>Subscribe For Latest Newsletter</h3></label>

```

Comment:
Error: Element h3 not allowed as child of element label in this context. (Suppressing further errors from this subtree.)